### PR TITLE
Assembl UI update (Batch 1)

### PR DESCRIPTION
**Pull request by pascalhuynh**
First batch of Assembl UI update

### CODE REVIEW

File: assembl/static2/js/app/components/debate/survey/proposals.jsx [hunk 1]
```diff
@@ -43,15 +43,18 @@ class Proposals extends React.Component {
...
-            <span
-              className={this.state.hideProposals ? 'assembl-icon-down-open color pointer' : 'assembl-icon-up-open color pointer'}
-              onClick={this.displayProposals}
-            />
-          </div>
-        </h3>
+        <div onClick={this.displayProposals} className="pointer">
+          <h3 className="collapsed-title padding-title">
+            <span className="title-txt">{questionTitle}</span>
+            <div className={postsToShow.length > 0 ? 'shown proposal-arrow' : 'hidden proposal-arrow'}>
```
Comment (paolina92) [inline assembl/static2/js/app/components/debate/survey/proposals.jsx]: You can use `import classnames from 'classnames'` but at this level of complexity, it's not very important

File: assembl/static2/js/app/components/debate/survey/proposals.jsx [hunk 2]
```diff
@@ -43,15 +43,18 @@ class Proposals extends React.Component {
...
+              <span
+                className={
+                  this.state.hideProposals ? 'assembl-icon-down-open color pointer' : 'assembl-icon-up-open color pointer'
```
Comment (paolina92) [inline assembl/static2/js/app/components/debate/survey/proposals.jsx]: Same here...

File: assembl/static2/js/app/components/debate/survey/proposals.jsx [hunk 3]
```diff
@@ -43,15 +43,18 @@ class Proposals extends React.Component {
...
+                }
+              />
+            </div>
+          </h3>
+        </div>
         {postsToShow.length > 0 ? (
           <div className={this.state.hideProposals ? 'hidden' : 'shown'}>
```
Comment (paolina92) [inline assembl/static2/js/app/components/debate/survey/proposals.jsx]: Same here...

File: assembl/static2/js/app/components/debate/survey/question.jsx [hunk 4]
```diff
@@ -134,15 +134,17 @@ export class Question extends React.Component<Props, State> {
...
-                  disabled={this.state.buttonDisabled}
-                  className="button-submit button-dark right margin-l clear"
-                >
-                  <Translate value="debate.survey.submit" />
-                </Button>
-              )}
+              <Button
+                onClick={this.createPost}
+                disabled={this.getPostBodyCharCount() > MINIMUM_BODY_LENGTH ? this.state.buttonDisabled : true}
+                className={
```
Comment (paolina92) [inline assembl/static2/js/app/components/debate/survey/question.jsx]: You should use `import classnames from 'classnames'` in this case

**Comment by pascalhuynh** [comment]
Thanks @paolina92 @MilanBarande !

**Pull request merged**